src/components/pages/search-algolia/Home/Home.jsx

```diff
@@ -118,14 +124,18 @@ export class Home extends PureComponent {
 
 Home.propTypes = {
   categoryCriterion: PropTypes.shape({
-    label: PropTypes.string.isRequired,
     icon: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
   }).isRequired,
   geolocationCriterion: PropTypes.shape({
     params: PropTypes.shape({
-      label: PropTypes.string.isRequired,
       icon: PropTypes.string.isRequired,
+      label: PropTypes.string.isRequired,
     }),
   }).isRequired,
   history: PropTypes.shape({ push: PropTypes.func }).isRequired,
+  sortCriterion: PropTypes.shape({
```

Comment:
Est-ce que ça vaut le coup de créer un objet Criterion qui serait utilisé ici et au dessus ?
```suggestion
  sortCriterion: PropTypes.instanceof(Criterion)
```

Reply:
👎